packages/foreman/rubygem-mime-types-data/rubygem-mime-types-data.spec

```diff
@@ -0,0 +1,93 @@
+# Generated from mime-types-data-3.2018.0812.gem by gem2rpm -*- rpm-spec -*-
```

Comment:
we already have mime-types-data in the Rails SCL: https://github.com/theforeman/rails-packaging/tree/tfm-ror52/packages/tfm-ror52-rubygem-mime-types-data